schema/project-level/project-schema.json

```
@@ -85,6 +85,14 @@
       "codelist": "projectType.csv",
       "openCodelist": false
     },
+    "relatedProjects": {
+      "title": "Related projects",
+      "description": "The details of projects related by being involved with the same infrastructure as this project. For example, a related project represented under a separate Open Contracting for Infrastructure Data Standard project identifier (OC4IDS id).",
```

Align language with definition of `project` and make example real-world:

```suggestion
      "description": "The details of projects related to the same set of infrastructure assets as this project. For example, to link a project for the replacement of a bridge to the project for its initial construction.",
```